CMakeLists.txt

```diff
@@ -324,6 +274,16 @@ ELSE(CMAKE_TOOLCHAIN_FILE)
     MESSAGE(WARNING "No default file for compiler (${COMPILER})")
   ENDIF()
 
+  # requires C++11 standard
+  SET(CMAKE_CXX_STANDARD 11)
+  SET(CMAKE_CXX_STANDARD_REQUIRED ON)
+  SET(CXX11_FLAG "-std=c++11")
```

Comment:
This code should not be necessary - the CMAKE_CXX_STANDARD variable should cause the correct flag to be set.

Reply:
I agree with you but that was not happening. So I leave the problem to @PDoakORNL . At the moment, you may get redundant flags.